src/zenml/environment.py

```diff
@@ -118,9 +119,19 @@ def in_notebook() -> bool:
             # We do not even have IPython installed
             return False
 
+    @staticmethod
+    def in_ipython_terminal() -> bool:
```

Comment:
My feedback here would be: Do we need to specifically differentiate between ipython REPL and normal python REPL? I think they should be treated the same to the end user, but the important thing is that they are treated differently from notebook/colab (because of the inlining of plots, etc).